### DISCUSSION TITLE
HUB75 with assigned E pin.

### DISCUSSION THREAD

**Discussion by ergindemir** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#268] [Q&A] [answered]
I am designing a board compatible with both HUB75 and HUB75E.
The E pin of the HUB75 connector is always grounded. 
Would this cause an issue with ESP32, when one of the pins is permanently grounded?
Would you suggest a pin that would be safe to ground? GPIO12 is low by default at boot, is it a good idea?
Thanks.

**Comment by vortigont** [comment, marked as answer]
Should not be any issues till you keep default config for the E pin (-1). Otherwise reconfigure pin to INPUT mode manually.
